docker-compose.yml

```diff
@@ -1,51 +1,44 @@
-fs-share:
+fs:
   image: tianon/true
   volumes:
-    - ./mysql-datadir:/var/lib/mysql:z
-    - ./mysql-shr:/shr:z
+    - ./mysql-datadir:/var/lib/mysql
+    - ./mysql-shr:/shr
     - ./mysql-autoload:/docker-entrypoint-initdb.d:ro
-    - ./mysql-conf.d:/etc/mysql/conf.d:z
-    - ./localdb-run.sh:/localdb-run.sh:rw
-    - ./deployments:/opt/jboss/wildfly/standalone/deployments:z
-    - ./wildfly-conf/configuration:/opt/keycloak/standalone/configuration:z
 
 fs-api:
   image: tianon/true
   volumes:
-    - ./nginx-conf:/etc/nginx/conf.d:rw
-    - ./nginx-certs:/etc/nginx/ssl:rw
-    - ./nginx-logs:/etc/nginx/logs:rw
+    - ./nginx-conf:/etc/nginx/conf.d
+    - ./nginx-certs:/etc/nginx/ssl
+    - ./nginx-logs:/etc/nginx/logs
 
 db:
-  image: mysql:latest
-  command: "sh /localdb-run.sh"
+  image: mysql
   environment:
     - TZ=Europe/Stockholm
     - MYSQL_ROOT_PASSWORD=password12
     - MYSQL_DATABASE=keycloak
     - MYSQL_USER=keycloak
     - MYSQL_PASSWORD=keycloak
   volumes_from:
-    - fs-share
+    - fs
 
 sso:
   image: dina/keycloak
   environment:
     - TZ=Europe/Stockholm
     - MYSQL_DATABASE=keycloak
-    - MYSQL_USER=keycloak
+    - MYSQL_USERNAME=keycloak
```

Comment:
That is from the last comment here: https://hub.docker.com/r/jboss/keycloak-mysql/. And it appeared it might not yet be fixed, looking at this: https://github.com/jboss-dockerfiles/keycloak/blob/master/server-mysql/changeDatabase.xsl, which is still using MYSQL_USERNAME. But not sure it is used or relevant.